modules/src/ics02_client/msgs.rs

```diff
@@ -15,17 +24,155 @@ pub enum ClientMsg {
 }
 
 /// A type of message that triggers the creation of a new on-chain (IBC) client.
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
It's usually a good practice to include the `Eq` bound when deriving `PartialEq`, unless there is a good reason not to do so (eg. equality is not an equivalence relation).

```suggestion
#[derive(Clone, Debug, PartialEq, Eq)]
```